configserver/src/main/java/com/yahoo/vespa/config/server/maintenance/SessionsMaintainer.java

```diff
@@ -35,5 +35,9 @@ protected void maintain() {
             int deleted = applicationRepository.deleteExpiredRemoteSessions(expiryTime);
             log.log(LogLevel.FINE, "Deleted " + deleted + " expired remote sessions, expiry time " + expiryTime);
         }
+
+        Duration lockExpiryTime = Duration.ofDays(1);
+        int deleted = applicationRepository.deleteExpiredLocks(lockExpiryTime);
+        log.log(LogLevel.INFO, "Deleted " + deleted + " expired locks, expiry time " + lockExpiryTime);
```

Comment:
nit: the middle part of the message could be simplified to " locks older than "

Reply:
thanks, will fix